assembly/src/parsers/stack_ops.rs

```diff
@@ -235,6 +235,21 @@ pub fn parse_swapw(span_ops: &mut Vec<Operation>, op: &Token) -> Result<(), Asse
     Ok(())
 }
 
+pub fn parse_swapdw(span_ops: &mut Vec<Operation>, op: &Token) -> Result<(), AssemblyError> {
```

Comment:
Let's add a short comment for this function (similar to how we do it for other functions here).